indexers/components/components.sql

```diff
@@ -1,22 +1,33 @@
 CREATE TABLE
   "versions" (
-    "id" SERIAL,
-    "block_height" BIGINT NOT NULL,
+    "id" SERIAL PRIMARY KEY,
+    "block_height" INT NOT NULL,
```

Comment:
I remember us running into issues using an `INT` instead of `BIGINT` here for `block_height`. Are we sure `INT` is within range? @gabehamilton

Reply:
"The maximum value for INT is 2,147,483,647 (2^31-1) while the maximum value for BIGINT is 9,223,372,036,854,775,807 (2^63-1)."

I dont understand the issue here. To my knowledge - The bigint type should only be used if the range of the integer type is insufficient, because the latter is definitely faster. We're saving 4bytes per entry with INT with our latest block height at 111,056,090 no where near the cap with over 2 billion leftover currently.

On the contrary I believe it could be argued that storage is cheap. The fire drill when you run out of integers is not cheap.

Reply:
It might appear that it will take > 60 years to reach MAX_INT block height (assuming 1 block per second), but in this case I would say it's best to use types that are as close to definitions as possible. The block height type is defined here: https://github.com/near/nearcore/blob/55f5586e89708f0b23f6a57aae58eaef270e77af/chain/indexer/src/lib.rs#L61 and I think it's best to use the same type here. Which is `BIGINT`.